backend/src/main/java/com/example/backend/data_access/stop/StopDAO.java

```diff
@@ -0,0 +1,216 @@
+package com.example.backend.data_access.stop;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import com.example.backend.data_access.*;
+import com.example.backend.data_access.route.RouteDAO;
+import com.example.backend.data_access.route.RouteDataAccessInterface;
+import com.example.backend.entity.Location;
+import com.example.backend.entity.Stop;
+import org.springframework.stereotype.Repository;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+@Repository
+public class StopDAO implements StopDataAccessInterface {
+
+    private final String stopCsvFilename = "src/data/stopData.csv";
```

Comment:
The filename should be changed to reflect the changes in directory structure.

Reply:
wdym ?